### PR TITLE
Fix the double-browser problem with dcos browse.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes https://github.com/Azure/azure-cli/issues/1649

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py [hunk 1]
```diff
@@ -156,8 +156,6 @@ def create_tunnel(self, remote_host, remote_port, local_port=0, open_url=None):
             remote_bind_address=(remote_host, remote_port),
             local_bind_address=('0.0.0.0', local_port)):
             try:
-                if open_url:
-                    webbrowser.open(open_url)
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py]: you could probably remove the webbrowser import

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -99,16 +100,19 @@ def acs_browse(resource_group, name, disable_browser=False):
     :type resource_group_name: String
     :param disable_browser: If true, don't launch a web browser after estabilishing the proxy
     :type disable_browser: bool
+    :param ssh_key_file: If set a path to an SSH key to use, only applies to DCOS [optional]
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: `[optional]` is redundant as unless a param is marked as `[Required]` it is optional.

Comment (brendandburns) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

**Comment by brendandburns** [comment]
Comment addressed, please re-check.

Thanks!

**Pull request merged**